src/app/App.ts

```diff
@@ -11,4 +12,5 @@ export const App = (): void => {
   const chessModel: ChessModel = new ChessEngine();
   const presenter = new ChessBoardPresenter(chessBoardView, chessModel);
   presenter.startGame();
+  pieceRendering();
```

Comment:
ta logike powinno realizowac showChessBoard z View. Ty nie wiesz kiedy to powinno byc wyswietlone itp. Wszystkim kieruje presenter w przypadku tej architektury. Bo tak np. wprowadzimy kolejny krok posredni, np. wysiwetlamy powitanie przed. No i presenter powinien miec cala logike kolejnosci wyswietlania itp :)

Reply:
tak tak, wczoraj mnie trochę zmogło i nie zdążyłem wszystkiego poprawić i pushnąć zmian. Skończę node to wracam do tego :-)